seatunnel-transforms/seatunnel-transforms-flink/seatunnel-transform-flink-split/src/main/java/org/apache/seatunnel/flink/transform/Split.java

```diff
@@ -94,6 +97,9 @@ public void prepare(FlinkEnvironment prepareEnv) {
         if (config.hasPath(SEPARATOR)) {
             separator = config.getString(SEPARATOR);
         }
+        if (config.hasPath(NAME)) {
+            name = config.getString(NAME);
+        }
```

Comment:
This is not referenced

Reply:
> default split function name = split,but we can define name in config file,like this:
 Split{
   separator=" "
   fields = ["word","name"]
   name = "splitx"
  }
  Split{
     separator="i"
     fields = ["word","test"]
     name = "splity"
    }

Reply:
ok, thanks your explain